src/rp2_common/pico_cyw43_driver/CMakeLists.txt

```diff
@@ -179,6 +179,16 @@ if (EXISTS ${PICO_CYW43_DRIVER_PATH}/${CYW43_DRIVER_TEST_FILE})
     # NO_EMBEDDED_PT argument (for example, if you need to chain into the binary, it
     # can't contain a partition table).
     #
+    # This will create a separate UF2 file for the firmware and a separate UF2 file for a
+    # TBYB version of the firmware. You will need to flash one of these to each device once,
```

Comment:
"...for the firmware..." and "...for a TBYB version of the firmware..." is pretty confusing. Is one of these the UF2 file containing the WiFi firmware? Is one of them a UF2 file containing the user's executable?
I agree with Graham that this needs to be made clearer.